api/app.js

```diff
@@ -69,31 +68,11 @@ app.use(['/api/household', '/api/households'], householdRouter);
 app.use(['/api/metric', '/api/metrics'], metricsRouter);
 app.use(['/api/location', '/api/locations'], locationRouter);
 
-// catch 404 and forward to error handler
-app.use(function (req, res, next) {
-  next(createError(404));
-});
-
-// error handler
-app.use(function (err, req, res, next) {
-  if (err instanceof createError.HttpError) {
-    res.locals.message = err.message;
-    res.locals.status = err.statusCode;
-    if (process.env.NODE_ENV === 'development') {
-      res.locals.error = err;
-    }
-  }
-  console.error(err);
-  if (process.env.NODE_ENV === 'production' && !res.locals.message) {
-    res.locals.message = 'ApplicationError';
-    res.locals.status = 500;
-  }
-  if (res.locals.status) {
-    res.status(res.locals.status || 500);
-    const errObject = { error: res.locals.error, message: res.locals.message };
-    return res.json(errObject);
-  }
-  next(err);
+app.use((err, req, res, next) => { // eslint-disable-line
+  res.status(err.status || 500).json({
+    message: err.message,
+    stack: err.stack,
```

Comment:
I love me some error handling!

In this case, showing the 'stack' in a production site can open it up to vulnerabilities.
Consider adding a check for if it's in production based on the env. 

Although I realize the frontend team is using a deployed backend that Heroku marks as production for their testing. So actually, maybe just leave a comment for the next cohort.

Reply:
Good catch. I wasn't sure if either the stack or message would be a problem, or both